LondonBritishLibrary/orient/BLorient8823.xml

```diff
@@ -0,0 +1,266 @@
+                    <history>
+                        <origin>
+                            <origDate notBefore="1700" notAfter="1799">18th century</origDate>
+                        </origin>
+                        <provenance>According to <bibl><ptr target="bm:Emmel2023Curzon"/><citedRange unit="page">236-241</citedRange>
+                            </bibl> acquired by <persName role="owner" ref="PRS3244Curzon"/>, whose name is inscribed inside the front cover, 
+                            during his stay in Egypt in <date notBefore="1837" notAfter="1838">1837-1838</date>. 
+                        </provenance>
+                        <acquisition>On <date when="1876-04-19">19 April 1876</date> - three years after the death of the owner
+                            <persName role="owner" ref="PRS3244Curzon"/>, the codex was given as a deposit on loan 
+                            to the British Museum (later: <placeName ref="INS00001BL">British Library</placeName>), then as a bequest from his 
+                            daughter <persName role="bequeather" ref="PRS14320DareaCurzon">Darea</persName> on 
+                            <date when="1917-10-10">10 October 1917</date> as inscribed in another note on the front flyleaf.</acquisition>
+                    </history>
+                    <additional>
+                        <adminInfo>
+                            <recordHist>
+                                <source>
+                                    <listBibl type="catalogue">
+                                        <bibl>
+                                            <ptr target="bm:Strelcyn1978BritishLibrary"/>
+                                            <citedRange unit="page">5-6</citedRange>
+                                        </bibl> 
+                                    </listBibl>
+                                    <listBibl type="secondary">
+                                        <bibl>
+                                            <ptr target="bm:Emmel2023Curzon"/>
+                                            <citedRange unit="page">236-241</citedRange>
+                                        </bibl>
+                                    </listBibl>
+                                </source>
+                            </recordHist>
+                        </adminInfo>
+                    </additional>
+                </msDesc>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="Apocrypha"/>
+                    <term key="Bible"/>
+                    <term key="ChristianContent"/>
+                    <term key="ChristianLiterature"/>
+                    <term key="OldTestament"/>
```

Comment:
BetaEsrael literature? "Literature" is a bit far fetched, but otherwise filtering for manuscripts related to the Beta ʾƎsrāʾel is difficult.